cmd/minikube/cmd/start.go

```diff
@@ -588,6 +588,7 @@ func selectDriver(oldConfig *cfg.Config) string {
 			out.ErrT(out.Documentation, "Documentation: {{.url}}", out.V{"url": st.Doc})
 		}
 		out.ErrLn("")
+		os.Exit(1)
```

Comment:
Let's collect more evidence before determining that every warning message should result in an exit. For instance: an older release of hyperkit is probably fine for many people.

I do agree for exiting early in the case of a driver that just isn't installed -- see #5840

Also, minikube should usually not force exits without consulting the `--force` variable.

Reply:
I agree I recommend not exiting... We don't have enough evidence it won't work for everyone. There are some evidence reinstallling docker machine fixes it.... But overall it's good to detect super old hyperkits and let user know of it

Reply:
Originally the purpose of this PR was FastFail, but I deleted this line because this is overlaps with # 5840.

Reply:
I left only the hyperkit version check logic.